test/index.test.js

```diff
@@ -0,0 +1,271 @@
+  {
+    id: 420,
+    url: 'https://www.tvmaze.com/episodes/420/grimm-1x07-let-your-hair-down',
+    name: 'Let Your Hair Down',
+    season: 1,
+    number: 7,
+    type: 'regular',
+    airdate: '2011-12-16',
+    airtime: '21:00',
+    airstamp: '2011-12-17T02:00:00+00:00',
+    runtime: 60,
+    rating: {
+      average: 8.2,
+    },
+    image: {
+      medium: 'https://static.tvmaze.com/uploads/images/medium_landscape/15/39292.jpg',
+      original: 'https://static.tvmaze.com/uploads/images/original_untouched/15/39292.jpg',
+    },
+    summary: '<p>Mysterious homicide deep in the woods leads to Nick and Hank reopening a long-aborted missing persons case. When Nick suspects this abducted person might be a feral creature, he asks Monroe to help remind this wild child of her human nature.</p>',
+    _links: {
+      self: {
+        href: 'https://api.tvmaze.com/episodes/420',
+      },
+    },
+  },
+];
+
+const createElement = () => {
+  document.body.innerHTML = '<h2>Search Results()</h2><main class="cards"></main>';
+  const searchResults = document.querySelector('h2');
+  const cards = document.querySelector('.cards');
+  const dataArray = searchQuery;
+  let searchCount = 0;
+  dataArray.forEach((el) => {
+    const div = document.createElement('div');
+    div.classList.add('cardItem');
+    const divImg = document.createElement('div');
+    divImg.classList.add('cardImg');
+    divImg.style.backgroundImage = `url(${el.image.original})`;
+    const h1 = document.createElement('h1');
+    h1.classList.add('cardName');
+    h1.textContent = `S${el.season}E${el.number} ${el.name}`;
+    const details = document.createElement('p');
+    details.classList.add('cardDetails');
+    details.innerHTML = `Plot Summary: <br>${el.summary}`;
+    const h2 = document.createElement('h2');
+    h2.classList.add('cardRuntime');
+    h2.textContent = `Runtime: ${el.runtime} mins Rating: ${el.rating.average}`;
+
+    const starContainer = document.createElement('div');
+    starContainer.classList.add('starContainer');
+
+    const starRate = document.createElement('span');
+    starRate.classList.add('material-icons-round');
+    starRate.classList.add('icons');
+    starRate.classList.add('starRate');
+    starRate.textContent = 'star_rate';
+
+    const starCount = document.createElement('span');
+    starCount.classList.add('starCount');
+    starCount.setAttribute('id', el.id);
+    starCount.textContent = '0';
+
+    const starBorder = document.createElement('span');
+    starBorder.classList.add('material-icons-round');
+    starBorder.classList.add('icons');
+    starBorder.classList.add('starBorder');
+    starBorder.textContent = 'star_border';
+    starBorder.setAttribute('id', el.id);
+
+    // Like Event
+    // starBorder.addEventListener('click', () => {
+    //   postLike(el.show.id);
+    //   starBorder.classList.toggle('liked');
+    //   starCount.setAttribute('disabled', true);
+    //   setTimeout(updateLikes, 1000);
+    // });
```

Comment:
- [ ] @ team as indicated earlier its not good practice to have commented code in your codebase.